docs/release_notes/2024/August_2024/August_2024.md

```diff
@@ -9,31 +9,73 @@ tags:
   - august
 ---
 
+## 3.3.10.\* (August 30, 2024)
+
+- Prophecy Python libs version: 1.9.9
+- Prophecy Scala libs version: 8.0.31
+
+### Features {#Features3310}
+
+#### Spark Copilot Enhancements {#Spark3310}
+
+- **Source suggestion**: While working on your Pipeline, if there is an unconnected source, Data Copilot will now suggest other sources that you can add to it on the graph.
```

Comment:
I don't quite understand how this is different than the current "suggested source"

Reply:
This feature was earlier present in SQL only. Now it is extended to Spark projects as well.